diagrams-pandoc.cabal

```diff
@@ -51,7 +52,7 @@ test-suite test
   main-is:             Main.hs
   build-depends:       diagrams-pandoc,
                        bytestring >= 0.10.10 && < 0.13,
-                       pandoc >= 2.19.2 && < 3.2,
+                       pandoc >= 3.0 && < 3.2,
```

Comment:
We need `pandoc >= 3.0` and `pandoc-types >= 1.23` now, since we make use of the quite new `Figure` construct of Pandoc AST. Fortunately these versions seem to support the GHC versions we want to support, as the CI reports.